test/expect/TestCustomOperators.test_script_graph_contains_custom_op.expect

```diff
@@ -1,4 +1,4 @@
 graph(%x : Dynamic) {
-  %1 : Dynamic = ^aten::relu()(%x)
+  %1 : Dynamic = aten::relu(%x)
```

Comment:
How does this one work? Does this override the original ATen op? There's `aten::relu` with exactly the same signature as this one.

Reply:
I suppose this is a non-standard example because no one should be calling `torch.ops.aten.relu` (it's just the easiest way to test custom ops because they're already built in), but it would have the same effect as calling `torch.relu` in this case, which dispatches to the same `aten::relu`